src/scrape_up/MediEncyclopedia/mediencyclopedia.py

```diff
@@ -0,0 +1,72 @@
+from html.parser import HTMLParser
+import bs4
+from bs4 import BeautifulSoup
+import requests
+
+"""
+        Create an instance of `MediEncyclopedia` class.
+
+    ```python
+    ency = MediEncyclopedia()
+    ```
+
+    | Methods               | Details                                                             |
+    | --------------------- | ------------------------------------------------------------------- |
+    | `.scrapebyurl()`      | Returns the medical dictation of associated topic url               |
+    | `.byletter()`         | Returns the list of medical relics starting with a particular letter|
+    
+
+    """
+
+
+class MediEncyclopedia:
+
+  def __init__(self):
+    print(self)
+
+  def scrapebyurl(self,url):
```

Comment:
Find another method to execute this program instead of asking them for the URL.

Reply:
I've added a new method query(userquery), This function takes a string userquery from the users and returns all the matching terms/responses from the encyclopedia matching the user's query. If there is no matching response, than it returns None.
    Parameters: userquery(string): It is used for taking a query input from the user.

Reply:
You can test the new method using the following notebook: https://colab.research.google.com/drive/1w8OiWjIi4XYnX3YVNjML6LAcIBrXtJuv?usp=sharing